src/redux/actions/TraineeAction.ts

```diff
@@ -0,0 +1,78 @@
+import creator from "./creator";
+import { GET_TRAINEE, CREATE_TRAINEES} from "..";
+import axios from "axios";
+export const getAllTraineess = ({ page,itemsPerPage,  All }:any) => async (dispatch: any) => {
+  try {
+    const datas = await axios({
+      url: "http://localhost:4000/",
+      method: "post",
+      data: {
+        query: `
+        query AllTraineesAttribute($input: pagination) {
```

Comment:
did you use the curren**t develop branch of the backend** because  **this query is not in the schema** and tried to change it to the right one and got some couple errors

Reply:
It is a query Fx solving the server api error #18

Reply:
that branch is also not working well in fetching trainee attributes the last time that I checked